app/zpt/handler/geoloc_tools.py

```diff
@@ -0,0 +1,34 @@
+import numpy as N
+
+
+def sph2cart(azimuth, elevation, radius):
+    rcos_theta = radius * N.cos(elevation)
+    x_cart = rcos_theta * N.cos(azimuth)
+    y_cart = rcos_theta * N.sin(azimuth)
+    z_cart = radius * N.sin(elevation)
+    return x_cart, y_cart, z_cart
+
+
+def cart2sph(x_cart, y_cart, z_cart):
+    hypot_xy = N.hypot(x_cart, y_cart)
+    radius = N.hypot(hypot_xy, z_cart)
+    elevation = N.arctan2(z_cart, hypot_xy)
+    azimuth = N.arctan2(y_cart, x_cart)
+    return azimuth, elevation, radius
+
+
+def getscangeoloc(lat_start, lon_start, lat_end, lon_end):
```

Comment:
```suggestion
def get_scan_geoloc(lat_start, lon_start, lat_end, lon_end):
```
or maybe even `...geo_loc` – very hard to read function name!